references/detection/train_pytorch.py

```diff
@@ -77,16 +78,16 @@ def main(args):
 
     print(args)
 
+    if not isinstance(args.workers, int):
+        args.workers = min(16, mp.cpu_count())
+
     torch.backends.cudnn.benchmark = True
 
     st = time.time()
     val_set = DetectionDataset(
         img_folder=os.path.join(args.data_path, 'val'),
         label_folder=os.path.join(args.data_path, 'val_labels'),
-        sample_transforms=Compose([
-            Lambda(lambda x: x / 255),
```

Comment:
Why do you remove the normalization here ?

Reply:
The  `to_tensor` in the dataset `__getitem__` already does that :)